deploy/src/main/deploy/docker/swarm_deploy.sh

```diff
@@ -19,14 +19,27 @@ CONFIG=$SCRIPTPATH/../../config
 CERTS=$CONFIG/hono-demo-certs-jar
 NS=hono
 CREATE_OPTIONS="-l project=$NS --network $NS --detach=false"
+if [ -z "$1" ] ; then
```

Comment:
can we do a check for something meaningful, e.g. `--no-adapters`?

Reply:
Sure, if you also think it makes sense to have a "no adapter" deployment, then we can adapt this idea.